src/components/Button.tsx

```diff
@@ -40,13 +68,16 @@ const linkStyles = css`
     }
 `;
 
+type ButtonPriority = 'primary' | 'secondary';
+
 type Props = {
     url: string;
     linkText: string;
+    priority: ButtonPriority;
```

Comment:
I have a feeling we could make priority optional, initialising this to 'primary'. Might suit most use cases and the simpler we make this sort of generic components, the better.